rusoto/core/src/request.rs

```diff
@@ -40,7 +41,7 @@ pub struct HttpResponse {
     /// Status code of HTTP Request
     pub status: StatusCode,
     /// Contents of Response
-    pub body: Box<Read>,
+    pub body: Box<Stream<Item=Vec<u8>, Error=HttpDispatchError> + Send>,
```

Comment:
I'm not clear on the behavior here: does the redefinition of `body` mean it will use a `Vec<u8>` internally?  We purposefully hid the contents behind the `Read` trait to allow streaming reads from services.  An example is downloading a multi-GB file from S3: we don't want to load the whole thing into memory.

Reply:
`Item=Vec<u8>` here means that the stream delivers "chunks" of data in the form of `Vec<u8>`. Think of this as similar to an `Iterator<Item=Vec<u8>>`, except asynchronously.